pkg/cmd/pr/merge/merge.go

```diff
@@ -211,6 +211,8 @@ func mergeRun(opts *MergeOptions) error {
 	if reason := blockedReason(pr.MergeStateStatus, opts.UseAdmin); !opts.AutoMergeEnable && !isPRAlreadyMerged && reason != "" {
 		fmt.Fprintf(opts.IO.ErrOut, "%s Pull request #%d is not mergeable: %s.\n", cs.FailureIcon(), pr.Number, reason)
 		fmt.Fprintf(opts.IO.ErrOut, "To have the pull request merged after all the requirements have been met, add the `--auto` flag.\n")
+		mergeConflictLink := fmt.Sprintf("https://github.com/%s/%s/pull/%d/conflicts.\n", baseRepo.RepoOwner(), baseRepo.RepoName(), pr.Number)
```

Comment:
The pull request could also exist on a GHE instance so hard coding `https://github.com/` is not going to work here. We can use `baseRepo.RepoHost()` to solve this issue.